core/client/fs/src/test/java/alluxio/client/metrics/MetricsHeartbeatContextTest.java

```diff
@@ -42,7 +42,9 @@
   @Test
   public void testExecutorInitialized() {
 
-    ClientContext ctx = ClientContext.create();
+    InstancedConfiguration conf = ConfigurationTestUtils.defaults();
+    conf.set(PropertyKey.USER_RPC_RETRY_MAX_DURATION, "1s");
```

Comment:
Do we know why these tests "work" without these changes?

Reply:
The machine is 32C/64T, so my first guess is that there is some kind of race which is more easily reproducible on a high-core count machine.

I think the Powermock thing is a bug with powermock and might have something to do with the JVM? There is a very similar issue report here: https://github.com/powermock/powermock/issues/370